examples/coaxial_torus_config.json

```diff
@@ -0,0 +1,118 @@
+{
+    "name": "Coaxial Torus",
+    "units": {
+        "length": "mm",
+        "angle": "deg",
+        "potential": "V",
+        "temperature": "K"
+    },
+    "grid": {
+        "coordinates": "cylindrical",
+        "axes": {
+            "r": {
+                "to": 20.0,
+                "boundaries": "inf"
+            },
+            "phi": {
+                "from": 0,
+                "to":0,
+                "boundaries": "periodic"
+            },
+            "z": {
+                "from": -10,
+                "to": 10.0,
+                "boundaries": {
+                    "left": "inf",
+                    "right": "inf"
+                }
+            }
+        }
+    },
+    "medium": "vacuum",
+    "objects": [
+        {
+            "type": "semiconductor",
+            "material": "HPGe",
+            "bulk_type": "p",
+            "temperature": 77.0,
+            "charge_density_model": {
+                "name": "linear",
+                "r": {
+                    "init": 0,
+                    "gradient": 0
+                },
+                "phi": {
+                    "init": 0.0,
+                    "gradient": 0.0
+                },
+                "z": {
+                    "init": -1e7,
+                    "gradient": 0.0
+                }
+            },
+            "geometry": {
+                "type": "torus",
+                "c": 10.0,
+                "a": {
+                    "from": 2.0,
+                    "to": 5.0
+                },
```

Comment:
Can you explain me what `c` and `a` is? I expected something like `r1` and `r2`?

Reply:
`c`(radius from the center of the hole to the center of the torus tube) and `a`(tube radius) are described in lines 23 and 47 respectively. I reviewed some literature to see what is commonly used and ended up picking the notation from [wolfram](https://mathworld.wolfram.com/Torus.html).  I think we should not use `r1` and `r2` because it could get confused with coordinate `r`

Reply:
Is that not precisely why we should call it something like `r1` and `r2`  because it belongs to the `r` direction of the grid. I would prefer something expressive, `a`, `b`, and `c` are very generic and you have to consult a manual to find out what to put there.

Reply:
`r_1` I guess would but then `r_2` would not. How about `r_tube` and `r_center`?

Reply:
So in general try to go along with OpenSCAD / OpenCSG / Geant4. 
And I strongly prefer something with `r` at these are radii. 
In the config file we can also allow for different notations. Like also r_center and r_tube.
But please add also a documentation to it on top of the struct, like:
```julia
"""
    # struct Torus{T}

`r_1`: ....
`r_2`: ...
...
"""
struct Torus....
    ....
end
```

I am also suspicious about the `a_interval`?

Reply:
`a_interval` would be converted to `r_tube_interval` and `c` to `r_center`

Reply:
I don't understand how a Torus has a fixed interval in `r` as this interval depends on `z`.
Or am I missing something?

Reply:
Ah, is it that the torus can have a "hole". So like a tube and not a cylinder?

Reply:
Yes, I believe that's how it works!
![torus](https://user-images.githubusercontent.com/6920236/99795420-0f4a1200-2b2c-11eb-8c62-4609703658cf.png)